From d4535c0fc6afaff1525bf618031a7fcd2080b5ed Mon Sep 17 00:00:00 2001
From: Vitaly Kuznetsov <vkuznets@redhat.com>
Date: Thu, 29 May 2014 16:49:37 -0400
Subject: [virt] xen/spinlock: Don't use __initdate for xen_pv_spin

Message-id: <1401382177-12769-7-git-send-email-vkuznets@redhat.com>
Patchwork-id: 81162
O-Subject: [RHEL7.1 PATCH 6/6] xen/spinlock: Don't use __initdate for xen_pv_spin
Bugzilla: 1006977
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1006977
Upstream commit: c3b7cb1fd8c1513be99fb3cfb7f39c5116d80dac
Brew build: https://brewweb.devel.redhat.com/taskinfo?taskID=7482680

Author: Konrad Rzeszutek Wilk <konrad.wilk@oracle.com>
Date:   Mon Sep 9 13:08:49 2013 -0400

As we get compile warnings about .init.data being
used by non-init functions.

Reported-by: kbuild test robot <fengguang.wu@intel.com>
Signed-off-by: Konrad Rzeszutek Wilk <konrad.wilk@oracle.com>
Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/xen/spinlock.c b/arch/x86/xen/spinlock.c
index b3c1ee8..253f63f 100644
--- a/arch/x86/xen/spinlock.c
+++ b/arch/x86/xen/spinlock.c
@@ -105,7 +105,7 @@ static DEFINE_PER_CPU(char *, irq_name);
 static DEFINE_PER_CPU(struct xen_lock_waiting, lock_waiting);
 static cpumask_t waiting_cpus;
 
-static bool xen_pvspin __initdata = true;
+static bool xen_pvspin = true;
 static void xen_lock_spinning(struct arch_spinlock *lock, __ticket_t want)
 {
  int irq = __this_cpu_read(lock_kicker_irq);
-- 
1.7.1